config/dns.py

```diff
@@ -0,0 +1,92 @@
+
+import click
+from swsscommon.swsscommon import ConfigDBConnector
+import ipaddress
+
+
+NAMESERVERS_MAX_NUM = 3
+
+
+def is_valid_ip_address(address):
+    """Check if the given IP address is valid"""
+    try:
+        ip = ipaddress.ip_address(address)
+        if ip.is_reserved or ip.is_multicast or ip.is_loopback:
+            return
+
+        invalid_ips = [
+            ipaddress.IPv4Address('0.0.0.0'),
+            ipaddress.IPv4Address('255.255.255.255'),
+            ipaddress.IPv6Address("0::0"),
+            ipaddress.IPv6Address("0::1")
+            ]
+        if ip in invalid_ips:
+            return
+
+        return ip
+    except Exception:
```

Comment:
No, this function is used for IP address validation. The caller will check the return, and if the returned value is None it will log the correct error message based on the context.